client/webpack.config.js

```diff
@@ -56,6 +56,31 @@ const defaultBrowserslistEnv = isCalypsoClient && ! isDesktop ? 'evergreen' : 'd
 const browserslistEnv = process.env.BROWSERSLIST_ENV || defaultBrowserslistEnv;
 const extraPath = browserslistEnv === 'defaults' ? 'fallback' : browserslistEnv;
 
+function filterEntrypoints( entrypoints ) {
+	if ( ! process.env.ENTRY_LIMIT ) {
+		return entrypoints;
+	}
+
+	const allowedEntrypoints = process.env.ENTRY_LIMIT.split( ',' );
+	// eslint-disable-next-line no-console
+	console.warn( '[entrylimit] Only building entrypoints %s', allowedEntrypoints.join( ', ' ) );
```

Comment:
Could we move this log line down and have it print the keys of `allowed`? I'm thinking it might be more helpful this way if I have a typo in one of the entry names for me to see that it's missing from this log line.

Reply:
go for it! I'm not entirely sure what you mean, but code would make it clear